src/storybook.js

```diff
@@ -14,6 +14,16 @@ var nodeStatic = require('node-static');
 var Promise = require('bluebird');
 
 var storybookObj;
+const VALIDPORTS = [
```

Comment:
Add a comment to explain that these are valid ports for Sauce Connect, and link to SC docs